src/Sarif.Viewer.VisualStudio.Core/CodeAnalysisResultManager.cs

```diff
@@ -1084,5 +1089,19 @@ internal static string GetSolutionPath(DTE2 dte, IVsFolderWorkspaceService works
 
             return solutionPath;
         }
+
+        internal void AddAllowedFileExtension(string fileExtension)
```

Comment:
>AddAllowedFileExtension [](http://example.com/codeflow?start=22&length=23)

can we test this and the following method?

Reply:
Yes, pls see the test: `GetStoredObject_PopuplateHashSet_WithCustomComparer`